packages/components/package.json

```diff
@@ -42,5 +42,10 @@
 		"clean": "npx rimraf dist types",
 		"prepublish": "npm run clean",
 		"prepare": "transpile && tsc && copy-assets"
+	},
+	"devDependencies": {
+		"@storybook/addon-actions": "^5.2.8",
+		"@storybook/preset-scss": "^1.0.2",
+		"@storybook/react": "^5.2.8"
```

Comment:
Packages in monorepos managed by Lerna should never have `devDependencies`. They should be all hoisted to the top-level `package.json`. See also the [Gutenberg setup](https://github.com/WordPress/gutenberg/blob/master/package.json#L76-L82) which does it the right way.

Reply:
Also, please pin the `@storybook/*` dependencies to exact versions and define a Storybook monorepo in the project's `renovate.json`. That tells Renovate to always file only one PR to do all `@storybook/*` updates at once.

Reply:
> See also the Gutenberg setup which does it the right way.

Gutenberg has setup Storybook across all of their packages. In contrast, in this PR I've only setup storybook for the `@automattic/components` package because I want to test the components in isolation from the rest of Calypso.

I put the dependencies in `packages/components/package.json` since following the above reasoning, the Storybook dependencies were only relevant for the `@automattic/components` package.

I can think of cases where `devDependencies` should avoiding hoisting are necessary, but this isn't one of them and I've updated the PR as requested.

Reply:
Lerna docs [recommend](https://github.com/lerna/lerna#common-devdependencies) hoisting dev dependencies to the top, it's not a Calypso-only convention.

The NPM script to run Storybook (`npm run components:storybook:start`) is also in the top-level `package.json` and relies on the `start-storybook` binary being present in `node_modules/.bin`. If you added the dependencies to the package and didn't duplicate them in the root, too, as you did, the binary would be in `packages/components/node_modules/.bin/start-storybook` and the top-level NPM wouldn't find it.